Libraries/Core/InitializeCore.js

```diff
@@ -38,9 +38,11 @@ if (global.window === undefined) {
 // Set up collections
 const _shouldPolyfillCollection = require('_shouldPolyfillES6Collection');
 if (_shouldPolyfillCollection('Map')) {
+  require('_wrapObjectFreezeAndFriends');
```

Comment:
I'm a little surprised that this bare module identifier just works, without `@providesModule _wrapObjectFreezeAndFriends` or something like that. Should I be worried?

Reply:
This was a change to Haste (or `jest-haste[-map]`) about a year ago. It is configured to use file base names as haste identifiers instead.